packages/core/theme-module/theme/pages/Home.vue

```diff
@@ -32,6 +32,8 @@
       image="/homepage/newsletter.jpg"
       class="call-to-action"
     />
+          test: {{ $t('welcome') }}
```

Comment:
Why so?

Reply:
because WHO said we have to test a lot, so...